### DISCUSSION TITLE
multiple "character views"

### DISCUSSION THREAD

**Discussion by nblog** [jovibor/HexCtrl#87] [Ideas]
is it possible to add more than one "character view"?

![image](https://github.com/jovibor/HexCtrl/assets/10218627/49c9b807-f870-40aa-9cf4-a1ac6aa64eb6)
![image](https://github.com/jovibor/HexCtrl/assets/10218627/03211205-2d2e-4131-924c-b8dcdd28e738)

**Comment by jovibor** [comment]
No, it's not possible.
At the moment only one view is supported.
